pom.xml

```diff
@@ -2089,6 +2089,9 @@
                         <injectAllReactorProjects>true</injectAllReactorProjects>
                         <!-- A workaround to make build work in a Git worktree, see https://github.com/git-commit-id/git-commit-id-maven-plugin/issues/215 -->
                         <useNativeGit>true</useNativeGit>
+                        <!-- git-commit-id 6.0.0 incorrectly detects the .git dir, breaking the native-git workaround; see https://github.com/git-commit-id/git-commit-id-maven-plugin/issues/639 -->
+                        <!-- Note that this is technically an internal detail (see MNG-6589) but will be publicly exposed under a new name in Maven 4.0.0 (see MNG-7038) -->
+                        <dotGitDirectory>${maven.multiModuleProjectDirectory}/.git</dotGitDirectory>
```

Comment:
We use the `air.main.basedir` property for this. Looks like we'll be able to drop that custom property with Maven 4, but let's keep using it for now.

Reply:
Nice, thanks!